### DISCUSSION TITLE
How do you export option attribute translations?

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#3387] [Questions & Answers] [answered]
As the title suggests, I ask this question with a practical example.

I have an OpenMage installation structured to be multisite, multistore and frontend multilanguage.

When I create a new website > shop > frontend/language I need to open custom attributes and translate all options.

I have attributes with more than 250 options, you understand by yourself that going to individual fields to copy and paste the translation is difficult.

besides the amount of rows there is also a huge amount of columns
(in my case 250 rows 30 columns) and the monitor does not allow you to have an overview.

with open mage import export tools i can't find the correct function (or setting) to create a bach to help me.

suggestions?

**Comment by kiatng** [comment]
Product attributes and custom options are two different things. In the database, attributes are managed in the eav attribute tables; custom options are managed in the product option tables. Each is handled very differently. So, which one are you talking about?

**Reply by Danieleeffe1** [reply]
sorry, you are right,

I'm talking about the attribute options

**Reply by kiatng** [reply, marked as answer]
I see, I think you are referring to a product attribute of type drop-down, and you want to export/import the translations of each of the drop-down option. The translation is by store. You can take a look at this table `eav_attribute_option_value`.
